src/signals/my-incidents/pages/BasePage.tsx

```diff
@@ -22,24 +29,34 @@ export const BasePage: FC<Props> = ({
   pageInfo,
   paragraphs,
 }) => {
-  const { documentTitle, dataTestId, pageTitle } = pageInfo
+  const { documentTitle, pageTitle } = pageInfo
 
   return (
-    <BasePageWrapper
-      documentTitle={documentTitle}
-      data-testid={dataTestId}
-      pageTitle={pageTitle}
-    >
-      {paragraphs &&
-        paragraphs.map((paragraph, index) => (
-          <Paragraph key={index} fontSize={16}>
-            {paragraph}
-          </Paragraph>
-        ))}
-
-      {children}
-
-      <ButtonWrapper>{buttons}</ButtonWrapper>
-    </BasePageWrapper>
+    <ContentWrapper>
+      <Helmet
+        defaultTitle={configuration.language.siteTitle}
+        titleTemplate={`${configuration.language.siteTitle} - %s`}
+      >
+        {documentTitle && <title>{documentTitle}</title>}
+      </Helmet>
+
+      <article>
+        {pageTitle && (
+          <header>
+            <StyledHeading>{pageTitle}</StyledHeading>
+          </header>
+        )}
+        {paragraphs &&
+          paragraphs.map((paragraph, index) => (
+            <Paragraph key={index} fontSize={16}>
```

Comment:
would be much better to make this 1 rem, and set the base font size to 16. there is is 27 matches for '16px'. 

https://datapunt.atlassian.net/browse/SIG-4889

tbc!

Reply:
Ja goeie!